boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -332,3 +333,27 @@ opm-build-push: docker-push
 .PHONY: ensure-fips
 ensure-fips:
 	${CONVENTION_DIR}/configure-fips.sh
+
+.PHONY: migrate-tools
+migrate-tools:
+	# Installing controller-gen v0.8.0
+	go install sigs.k8s.io/controller-tools/cmd/controller-gen@${CONTROLLER_GEN_VERSION}
+	mv ${GOBIN}/controller-gen ${GOBIN}/controller-gen-${CONTROLLER_GEN_VERSION}
+
+	# Installing openapi-gen v0.23.0
+	go install k8s.io/code-generator/cmd/openapi-gen@${OPENAPI_GEN_VERSION}
+	mv ${GOBIN}/openapi-gen ${GOBIN}/controller-gen-${CONTROLLER_GEN_VERSION}
+
+.PHONY: migrate-files
+# You will need to export the forked/cloned operator repository directory as REPO to make this work.
+# Example: export REPO=~/Projects/My-Operator-Fork
+migrate-files:
```

Comment:
Suggest naming this rule more descriptively, e.g. `migrate-osdk1-files`.

(Or if you take my other suggestions, you could just inline this stuff into the `migrate` rule :shrug:)

Reply:
Inlining sounds like the better option.